projects/js-packages/connection/components/connect-screen/required-plan/index.jsx

```diff
@@ -101,6 +114,10 @@ ConnectScreenRequiredPlan.propTypes = {
 	priceAfter: PropTypes.number.isRequired,
 	/** The Currency code, eg 'USD'. */
 	pricingCurrencyCode: PropTypes.string,
+	/** The WordPress.com product slug. If specified, the connection/authorization flow will go through the Checkout page for this product'. */
+	wpcomProductSlug: PropTypes.string,
```

Comment:
Could we call this `productSlug` instead?

Reply:
I don't think so because they reference different things. `productSlug` are the products that Jetpack offers, such as `Backup`, `Anti Spam`, `Search`, etc. `wpcomProductSlug` is the product name in the WordPress.com plans/products context. The `products` endpoint provides this data.

Reply:
I guess the concern Jason has here is that for search we have a WPCOM product and a Jetpack product, which are basically the same thing but for different types of sites. It could be ambiguous for search really.

But for products in general, I guess it doesn't have the issue. I'll keep the usage of `wpcomProductSlug`, however I think `productSlug` and `wpcomProductSlug` aren't really that different. `wpcomProductSlug` has its limitations too, like why wpcom not jetpack 😄 

addressed in 5328909d19